stats/lib/stats-fetchers.js

```diff
@@ -20,7 +20,10 @@ export const fetchRetrievalSuccessRate = async (pgPools, filter) => {
     day: r.day,
     total: r.total,
     successful: r.successful,
-    success_rate: r.total > 0 ? r.successful / r.total : null
+    success_rate: r.total > 0 ? r.successful / r.total : null,
+    successful_http: r.successful_http ?? null,
+    // successful_http might be null because the column was added later
+    success_rate_http: r.total > 0 && !(r.successful_http === undefined || r.successful_http === null) ? r.successful_http / r.total : null
```

Comment:
```suggestion
    success_rate_http: r.total > 0 && typeof r.successful_http === 'number' ? r.successful_http / r.total : null
```

I think this is simpler and still covers all cases

Reply:
Not sure if this works. successful_http is a string representation of a number.
We could use something like this for the check to be more concise and reuse the expression: 
`const isValidNumber = value => value != null && !isNaN(+value)`
And then call:
`    success_rate_http: r.total > 0 && isValidNumber(r.successful_http) ? r.successful_http / r.total : null
`

Reply:
FYI:
node-pg represents 64-bit integers coming from Postgres as JavaScript strings to preserve precision.

Even if the column `successful_http` is represented as a 32-bit integer, `SUM(successful_http)` yields a 64-bit integer.

We could cast `successful_http` to `INT` in the Postgres query so that we receive a Number on the JavaScript side, but that would create an inconsistency with how we handle `successful`, so this is not a good solution.

I think we can trust that the database returns either `null` (when the column value is not set) or a string representation of a number.

```suggestion
    success_rate_http: r.total > 0 && r.successful_http !== null ? r.successful_http / r.total : null
```

Reply:
Done!